src/app/view/components/dot-navigation/services/dot-navigation.service.ts

```diff
@@ -63,8 +66,14 @@ export class DotNavigationService {
         private dotRouterService: DotRouterService,
         private dotcmsEventsService: DotcmsEventsService,
         private loginService: LoginService,
-        private router: Router
+        private router: Router,
+        private dotLocalstorageService: DotLocalstorageService
     ) {
+        const savedMenuStatus = this.dotLocalstorageService.getItem<boolean>(DOTCMS_MENU_STATUS);
+        this._collapsed$.next(
+            savedMenuStatus === undefined || savedMenuStatus === null ? true : savedMenuStatus
```

Comment:
```suggestion
            savedMenuStatus === false ? false : true
```

Reply:
Can't because if is `undefined` (meaning that is not set in the localstorage) I have to set it a `true`, by default now the menu will be collapsed.